examples/platform_get_submission_count

```diff
@@ -0,0 +1,5 @@
+from metriq import MetriqClient
```

Comment:
This file appears to not have an extension. It should be a Python file, so it should be named `platform_get_submission_count.py`.

Reply:
My bad, missed that - fixed it!

Reply:
Nice. Thank you!